ui/perfherder/alerts/StatusDropdown.jsx

```diff
@@ -73,6 +78,41 @@ export default class StatusDropdown extends React.Component {
     };
   };
 
+  getNumberOfNonWorkingDays(createdAt, dueDate) {
+    let count = 0;
+    const curDate = new Date(createdAt.getTime());
```

Comment:
`curDate` is not suggesting that we are referring to the `createdAt` time and is confusing since it differs from the current date.

Reply:
`createdDate`